data/user_data.rb

```diff
@@ -0,0 +1,26 @@
+STARTER_PASSWORD ||= ENV["STARTER_PASSWORD"]
```

Comment:
This looks like a file I typically see in `db/seeds.rb`...a script to be run upon initializing the database.